.changeset/quick-lies-invent.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-docs/gatsby-theme-docs': minor
```

Comment:
If you are removing functionalities, you should bump this as `major`

```suggestion
'@commercetools-docs/gatsby-theme-docs': major
```

Reply:
@emmenko , thank you for the comment. This is going to be deployed to docs as part of an ongoing release which is already bumping a major version, technically this is a patch of the major. So no need to re-bump the major FYI @timonrey

Reply:
I understand but it's still good practice to follow semantic versioning correctly.

For instance, we have the changesets that provide a great way to document changes. In particular for breaking changes it allows to document things properly (including e.g. migration steps).

Having multiple changesets with major version bumps is also not a problem, as it actually allows to better differentiate the changes in the release.

Reply:
Makes sense.